core/src/test/java/nl/nn/adapterframework/http/rest/ApiListenerPatternsTest.java

```diff
@@ -20,19 +20,17 @@
 import java.util.Arrays;
 import java.util.Collection;
 
-import nl.nn.adapterframework.http.rest.ApiListener.HttpMethod;
-
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.MethodSource;
 
+import nl.nn.adapterframework.http.rest.ApiListener.HttpMethod;
+
 
 public class ApiListenerPatternsTest {
 
 	private ApiListener listener;
-	private String expectedUriPattern;
-	private String expectedCleanPattern;
 
-	public static Collection<Object[]> data() {
+  public static Collection<Object[]> data() {
```

Comment:
Weird indentation?

Reply:
Ja codacy valt er ook over

Reply:
Heb het aangepast.. maar is natuurlijk beetje jammer om daar weer 30m op te wachten ivm build. 
Dit gaat fout, met indentatie, als ik iets apply in mijn IDE. Sommige suggesties hebben een neven-effect. Ik zal kijken of ik format changed lines weer kan aanzetten oid, waardoor het auto gefixt wordt in de toekomst.

Reply:
Het gebeurt mij ook af en toe.